XamarinCommunityToolkit.UnitTests/ConvertersNamespace_Tests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using System.Reflection;
+using Xamarin.CommunityToolkit.Behaviors;
+using Xamarin.Forms;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests
+{
+	public class Namespace_Tests
+	{
+		[Fact]
+		public void MakeSureConvertersAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(BaseBehavior)).GetTypes();
```

Comment:
I guess would be slightly better to use some "converter" if one day we decide to split behaviors and converter into several assemblies.

Reply:
Good point! Done